src/applications/vaos/tests/new-appointment/components/CommunityCareProviderSelectionPage/index.unit.spec.jsx

```diff
@@ -293,7 +287,7 @@ describe('VAOS <CommunityCareProviderSelectionPage>', () => {
     userEvent.click(
       await screen.findByRole('button', { name: /Yes, remove provider/i }),
     );
-    expect(await screen.findByText(/Choose a provider/i));
+    expect(await screen.findByRole('button', { name: /Choose a provider/i }));
```

Comment:
Changed to findByRole since there where multiple values returned with findByText.